FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/FreeRTOSConfig.h

```diff
@@ -46,7 +46,7 @@
 #define configUSE_TICK_HOOK                      1
 #define configCPU_CLOCK_HZ                       ( ( unsigned long ) 25000000 )
 #define configTICK_RATE_HZ                       ( ( TickType_t ) 1000 )
-#define configMINIMAL_STACK_SIZE                 ( ( unsigned short ) 88 )
+#define configMINIMAL_STACK_SIZE                 ( ( unsigned short ) 96 )
```

Comment:
This looks odd. How about changing it to 128?

Reply:
Yes, 128 should be also a good number.

But 128 seems to trigger problems and I get a "malloc failed" output. I'll need a few days until I can look
into this again, seems there is another issue coming up here.

best regards,

Florian La Roche